components/advanced_create_comment/advanced_create_comment.tsx

```diff
@@ -413,9 +413,13 @@ class AdvancedCreateComment extends React.PureComponent<Props, State> {
         });
     }
 
-    toggleEmojiPicker = () => {
-        this.setState({showEmojiPicker: !this.state.showEmojiPicker});
-    }
+    toggleEmojiPicker = (e?: React.MouseEvent<HTMLButtonElement, MouseEvent>): void => {
+        if (e) {
+            e.stopPropagation();
+        }
```

Comment:
nit : Does this look ok?
```suggestion
e?.stopPropagation()
```

Reply:
as we dont do anything else with the `event` object anyways

Reply:
if you decide to change to the above, please do the changes in other places as well where we did 
```
if (e) {
            e.stopPropagation();
        }
```